build.gradle

```diff
@@ -63,7 +63,7 @@ subprojects {
   pluginManager.withPlugin("com.vanniktech.maven.publish") {
     mavenPublish {
       nexus {
-        groupId = "dev.zacsweers"
+        stagingProfile = "dev.zacsweers"
```

Comment:
There is also a property for that now `SONATYPE_STAGING_PROFILE `

Reply:
5615759